tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -135,4 +139,17 @@ func testFilesCopied(t *testing.T) {
 		assert.Emptyf(t, stdout.String(), "Missing file: %s", fullPath)
 	}
 
+	// Check the SHA of kube.tar.gz downloaded
+	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
+	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
+	stdout := new(bytes.Buffer)
+	_, err = client.Run(command, stdout, os.Stderr)
+	require.NoError(t, err, "Error generating SHA512 for %s", kubeTarPath)
+	// CertUtil output example:
+	// SHA512 hash of <filepath>:/r/n<SHA-output>/r/nCertUtil: -hashfile command completed successfully.
+	// Extracting SHA value from the output
+	actualSha := strings.Split(stdout.String(), "/r/n")[1]
```

Comment:
Check if we need to do something similar to this.

https://github.com/openshift/windows-machine-config-operator/pull/98/files#diff-1c2c1c572f356c7644a7be940f4d187dL131

Reply:
I can add the check to ensure kube.tar actually exists here before running the test for sha check